packages/asc-ui/src/components/TopTaskButton/TopTaskButtonStyle.ts

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import styled from '@datapunt/asc-core'
+import { themeSpacing, themeColor } from '../../utils'
+
+import IconStyle from '../Icon/IconStyle'
+
+export interface Props {
+  icon: React.ElementType
+  title?: string
+  content?: string
+  onDarkBackground?: boolean
+  width: string
+  className: string
+}
+
+const defaultProps = {
```

Comment:
Perhaps we can move the defaultProps to the .tsx file? Then we don't need to use it inside the styled-component.

Reply:
it issued for the width and onDarkBackground